Makefile

```diff
@@ -2,11 +2,11 @@ prefix ?= /usr/local
 bindir = $(prefix)/bin
 
 build:
-	swift build -c release --disable-sandbox
+	swift build -c release --disable-sandbox --arch arm64 --arch x86_64
 
 install: build
 	install -d "$(bindir)"
-	install ".build/release/xcresultparser" "$(bindir)"
+	install ".build/apple/Products/Release/xcresultparser" "$(bindir)"
```

Comment:
I believe when you configure swift build to create a universal binary it invokes the XCode build toolchain instead of using the built-in one, so the artifact gets generated in a different location.